### PR TITLE
Implement V3 redirect checkout

### PR DESCRIPTION
**Pull request by nicoletacristian**

### CODE REVIEW

File: src/SwedbankPay.Sdk/PaymentOrders/PaymentOrderRequest.cs [hunk 1]
```diff
@@ -27,10 +28,11 @@ public class PaymentOrderRequest
                                    Language language,
                                    bool generateRecurrenceToken,
                                    IUrls urls,
-                                   IPayeeInfo payeeInfo)
+                                   IPayeeInfo payeeInfo,
+                                   bool useCheckoutV3 = false)
```
Comment (asbjornu) [inline src/SwedbankPay.Sdk/PaymentOrders/PaymentOrderRequest.cs:32]: I feel that `bool useCheckoutV3` does not scale very well. What should we do when we launch Checkout v4? I would like to explore other designs for this, perhaps by adding a `PaymentOrderRequestBuilder` which can reduce the number of arguments in `PaymentOrderRequest` as well as make it immutable.

**Review by asbjornu — CHANGES_REQUESTED** [review]
Thank you so much for this PR! It's a great start, but we should probably think a bit more about how we want Checkout v3 implemented in the SDK. It may be worth doing some backwards incompatible changes for. Thoughts @zunkas?

File: src/SwedbankPay.Sdk/PaymentOrders/PaymentOrderRequest.cs (hunk 1 above)
Comment (mathiso) [reply src/SwedbankPay.Sdk/PaymentOrders/PaymentOrderRequest.cs:32]: Can you please just resolve this and complete this PR? Swedbank does not want us to use Checkout v2 any more.

**Comment by zunkas** [comment]
A new version of the SDK for version 3.1 have been merged to develop.

**Pull request closed without merging**